### ISSUE TITLE
Enhancement: more information about the text models via API

### ISSUE THREAD

**Issue by ceruleandeep** [Haidra-Org/AI-Horde#452]
If /v2/status/models?type=text returned more information about the models, clients e.g. SillyTavern could display it to end-users. Maybe other uses such as choosing instruct template or sampler defaults.

We got this request about the Horde model selector in ST: "Is there any way to get more details on Stable Horde models? Even a descriptive summary would help me decide which ones to try for a particular chat." But ST can only display what the Horde API provides.

There are some additional information in [db.json](https://raw.githubusercontent.com/db0/AI-Horde-text-model-reference/main/db.json) but it doesn't seem to be used in AI-Horde (or populated, probably because it's not used).

Perhaps a "metadata" dict could be added to /v2/status/models to give model name, description, URL, style, usage info, or whatever else people think could be helpful. I found the "Popular Models Listing" guide in KoboldAI Discord helpful so providing some of that information to frontends in a structured way would no doubt help more people.

**Comment by db0** [comment]
Yes the problem is exactly that the people who maintain the model reference do not provide details about the models.

**Comment by db0** [comment]
If you're interested in adding this information I can bring you in contact with the current maintainer to handle it

**Comment by ceruleandeep** [comment]
Thank you, that would be good. I did https://github.com/Haidra-Org/AI-Horde-text-model-reference/pull/78 as a proof of concept.
